modules/schema/src/test/java/org/apache/ignite/internal/schema/configuration/ConfigurationToSchemaDescriptorConverterTest.java

```diff
@@ -0,0 +1,308 @@
+            assertThat(((NumberNativeType) type).precision(), equalTo(typeView.precision()));
+        } else if (type instanceof DecimalNativeType) {
+            assertThat(((DecimalNativeType) type).precision(), equalTo(typeView.precision()));
+            assertThat(((DecimalNativeType) type).scale(), equalTo(typeView.scale()));
+        } else if (type instanceof TemporalNativeType) {
+            assertThat(((TemporalNativeType) type).precision(), equalTo(typeView.precision()));
+        } else if (type instanceof BitmaskNativeType) {
+            assertThat(((BitmaskNativeType) type).bits(), equalTo(TEST_LENGTH));
+        } else {
+            assertThat("Unknown type: " + type.getClass(), type.getClass(), equalTo(NativeType.class));
+        }
+    }
+
+    @ParameterizedTest
+    @MethodSource("generateTestArguments")
+    public void convertColumnViewConstantDefault(DefaultValueArg arg) {
+        String columnName = arg.type.spec().name();
+        ColumnTypeView typeView = TestColumnTypeView.forType(arg.type);
+        ColumnDefaultView defaultValueView = createViewForValue(arg.type, arg.defaultValue);
+        ColumnView columnView = new TestColumnView(columnName, typeView, defaultValueView);
+
+        Column column = ConfigurationToSchemaDescriptorConverter.convert(0, columnView);
+
+        assertThat(column.name(), equalTo(columnName));
+        assertThat(column.type(), equalTo(arg.type));
+        assertThat(column.nullable(), equalTo(arg.defaultValue == null));
+        assertThat(column.defaultValueProvider().type(), equalTo(Type.CONSTANT));
+        assertThat(column.defaultValue(), equalTo(arg.defaultValue));
+    }
+
+    @Test
+    public void convertColumnViewFunctionalDefault() {
+        String columnName = "UUID";
+        String functionName = DefaultValueGenerators.GEN_RANDOM_UUID;
+        ColumnTypeView typeView = TestColumnTypeView.forSpec(NativeTypeSpec.UUID);
+        ColumnDefaultView defaultValueView = new TestFunctionCallDefaultView(functionName);
+        ColumnView columnView = new TestColumnView(columnName, typeView, defaultValueView);
+
+        Column column = ConfigurationToSchemaDescriptorConverter.convert(0, columnView);
+
+        assertThat(column.name(), equalTo(columnName));
+        assertThat(column.type(), equalTo(NativeTypes.UUID));
+        assertThat(column.defaultValueProvider().type(), equalTo(Type.FUNCTIONAL));
+        assertThat(((FunctionalValueProvider) column.defaultValueProvider()).name(), equalTo(functionName));
+    }
+
+    @Test
+    public void convertTableView(@InjectConfiguration(
+            polymorphicExtensions = {
+                    UnknownDataStorageConfigurationSchema.class,
+                    NullValueDefaultConfigurationSchema.class,
+                    UnlimitedBudgetConfigurationSchema.class
+            },
+            value = "mock {"
+                    + "primaryKey {columns=[K1, K2], colocationColumns=[K2]},"
```

Comment:
SchemaValidationUtils#validateIndexes compares that all uniq fields contains in colocation fields, for test it is ok, but for cleanness - you can fix it )

Reply:
first of all I don't quite understand why did you mention indexes in this context. Second, there is no such class anymore :)